distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -21,8 +21,11 @@
 
 import java.io.IOException;
 import java.time.Duration;
+import java.util.Map;
 
+import org.apache.commons.lang.StringUtils;
 import org.glassfish.grizzly.http.server.Response;
+import org.glassfish.grizzly.http.util.HttpStatus;
 import org.slf4j.LoggerFactory;
 import org.slf4j.ext.XLogger;
```

Comment:
Fixed for `ChunkedContent`, `FederatedCswMockServer`, and `TestFederation` classes.